resources/aws_ec2.go

```diff
@@ -91,9 +91,11 @@ const (
 type awsEc2Event uint8
 
 const (
-	awsEc2EventWatchReady awsEc2Event = iota
+	awsEc2EventNone awsEc2Event = iota
```

Comment:
What do we need a `None` for?

Reply:
This is used as a zero value in the case that snsProcessEvent returns an error, or finds that the instance id does not match the resource we are watching. If we receive awsEc2EventNone, we can ignore the event.